### PR TITLE
Add block input events

### PR DESCRIPTION
**Pull request by Jiacomina**

### CODE REVIEW

File: assets/Script/menuScript.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ cc.Class({
   },
 
   // use this for initialization
-  onLoad() {
+  onLoad() { O
```
Comment (Jesse-Cameron) [inline assets/Script/menuScript.js]: I think you left a typo in here @Jiacomina 😹

File: assets/Script/technology/techGame1.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    const newPosition = cc.v2(nextX, that.block1.position.y);
+
+    const worldX = that.block1.parent.convertToWorldSpaceAR(newPosition).x;
+
+    if (worldX > 760) {
+      that.block1.emit('touchend');
+      return;
+    }
+
+    if (worldX < 160) {
```
Comment (Jesse-Cameron) [inline assets/Script/technology/techGame1.js]: Might be worth dynamically getting the left and right limits from the game frame.
It'll look something like:
```
const bufferSize = 200;
const leftBuffer = 0 + bufferSize;
const rightBuffer = cc.view.getFrameSize() - bufferSize;
```
I think

Comment (Jesse-Cameron) [reply assets/Script/technology/techGame1.js]: It'll have to be done in the `setupEventHandlers` scope as well.

**Review by Jesse-Cameron — APPROVED** [review]
Looking really good mate!
I think that you just need to rebase/merge from develop once more (sorry I merged in a feature).
And then add get the linter to pass (try running `npm run lint` or `npm run lint:fix`)

File: assets/Script/menuScript.js (hunk 1 above)
Comment (Jiacomina) [reply assets/Script/menuScript.js]: ahah oops

File: assets/Script/technology/techGame1.js (hunk 2 above)
Comment (Jiacomina) [reply assets/Script/technology/techGame1.js]: Ok, I've added frameSize adjustment and a constant buffer size value.

**Pull request merged**